Makefile

```diff
@@ -12,11 +12,26 @@ CLOUDFLARE_ENDPOINT ?= https://07be8d2fbc940503ca1be344714cb0d1.r2.cloudflaresto
 CLOUDFLARE_BUCKET ?= dist-staging
 CLOUDFLARE_PROFILE ?= worker
 LOGLEVEL ?= silent
-OSTYPE := $(shell uname -s | tr '[:upper:]' '[:lower:]')
-ifeq ($(findstring os/390,$OSTYPE),os/390)
-OSTYPE ?= os390
+ifeq ($(OS),Windows_NT)
+  OSTYPE := windows
+
+  ifeq ($(PROCESSOR_ARCHITEW6432),AMD64)
```

Comment:
This looks wrong to me. You compare against AMD64 and if so, you set arm64, and so forth.

Reply:
I think that this was meant to be ARM64?